src/server/connectors/config-db/config-db.connector.test.js

```diff
@@ -193,4 +193,23 @@ describe("Function: getLocalCouncils", () => {
       await expect(getLocalCouncils()).resolves.toEqual(localCouncilsMock);
     });
   });
+
+  describe("given the request is successful and some council URLs are NULL or empty", () => {
+    const localCouncilsMock = ["", "cardiff", null, "the-vale-of-glamorgan"];
+    beforeEach(() => {
+      mongodb.MongoClient.connect.mockImplementation(() => ({
+        db: () => ({
+          collection: () => ({
+            distinct: () => localCouncilsMock
+          })
+        })
+      }));
+    });
+
+    it("should return the list of councils from distinct() response and not contain any NULL or empty values", async () => {
+      await expect(getLocalCouncils()).resolves.toEqual(
+        expect.not.arrayContaining([null, ""])
```

Comment:
I think you should check what is returned as well as what's not - from a whitebox testing perspective your tests is fine since you already checked returned values above but I think it's worth a quick check